StructureDefinition/Extension-DM-prescriptionType.StructureDefinition.xml

```diff
@@ -1,9 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <StructureDefinition xmlns="http://hl7.org/fhir">
-  <id value="8e2143ee-5d67-4731-9b23-6dc3aa72f9ae"/>
-  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionType" />
-  <name value="ExtensionNHSBSAprescriptionType" />
+  <id value="Extension-England-DMPrescriptionType" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-DMPrescriptionType" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandNHSBSAPrescriptionType" />
```

Comment:
Name and Title mention NHSBSA but not in ID. NHSBSA should be dropped from Name/Title to be consistent with URL/ID.

Reply:
Did the required changes.